build.gradle

```diff
@@ -620,6 +620,11 @@ project('test:system') {
     // Specifically publish this test project
     apply plugin: 'maven'
 
+    // override the compile java options to disable -Werror. This will be removed once https://github.com/kubernetes-client/java/issues/436
+    compileJava {
+        options.compilerArgs.remove("-Werror")
```

Comment:
Will this suppress only the warnings you are referencing in the comment or any possible warning?